include/OpenSpaceToolkit/Physics/Time/Time.hpp

```diff
@@ -239,6 +259,28 @@ class Time
 
     static Time Parse(const String& aString, const Time::Format& aFormat = Time::Format::Undefined);
 
+    /// @brief            Constructs a time from a real number of hours
+    ///
+    /// @code
+    ///                   Time time = Time::Hours(12.567) ; // 12:34:01.199.999.999
+    /// @endcode
+    ///
+    /// @param            [in] aReal A real number of hours
+    /// @return           Time
+
+    static Time Hours(const Real& aReal);
+
+    /// @brief            Constructs a time from a real number of seconds
+    ///
+    /// @code
+    ///                   Time time = Time::Seconds(12.567) ; // 12:34:01.199.999.999
+    /// @endcode
+    ///
+    /// @param            [in] aReal A real number of seconds
+    /// @return           Time
+
+    static Time Seconds(const Real& aReal);
```

Comment:
```suggestion
    static Time DecimalSeconds(const Real& aReal);
```
Perhaps?

Reply:
Hmmmmm I think the signature and the docstring make it sufficiently obvious?

Reply:
I think it's clear from the signature